tests/unit_tests/test_json_utilities.py

```diff
@@ -0,0 +1,37 @@
+"""Unit tests for json_unitilies."""
+
+import json
+import numpy as np
+from emodelrunner.json_utilities import NpEncoder
+
+
+def test_np_encoder_array():
+    """Unit test for the NpEncoder on np.array."""
+    test_np_array = np.array([1, 2, 3])
+    test_np_array_encoded = json.dumps(test_np_array, cls=NpEncoder)
+    test_np_array_decoded = json.loads(test_np_array_encoded)
+    assert test_np_array_decoded == test_np_array.tolist()
+
+
+def test_np_encoder_integer():
+    """Unit test for the NpEncoder on np.integer."""
+    test_np_integer = np.int16(1)
+    test_np_integer_encoded = json.dumps(test_np_integer, cls=NpEncoder)
+    test_np_integer_decoded = json.loads(test_np_integer_encoded)
+    assert test_np_integer_decoded == test_np_integer
+
```

Comment:
Very nice! Could also add a similar test function for numpy floats, like np.float32, for coverage.

Reply:
Thanks :)

Reply:
added in the next commit